Consul/Tests/Services/AbstractTest.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace SensioLabs\Consul\Tests\Services;
+
+use Nekland\Tools\StringTools;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Process\Process;
+
+abstract class AbstractTest extends \PHPUnit_Framework_TestCase
+{
+    private static $consul;
+
+    public static function setUpBeforeClass()
+    {
+        static::clean(true);
+        $os = PHP_OS === 'Darwin' ? 'mac' : 'linux';
+        self::$consul = new Process('exec ' . __DIR__ . '/../../../bin/consul/consul_' . $os . ' agent -dev -bind=127.0.0.1');
+        self::$consul->start();
+
+        // This would be really better with https://github.com/symfony/symfony/pull/27742
+        $wait = true;
+        do {
+            self::$consul->checkTimeout();
+            $output = self::$consul->getOutput();
+
+            if (StringTools::contains($output, 'Synced node info')) {
```

Comment:
I'm :-1: for adding a dep for only replacing a strpos.

Reply:
It's actually more: https://github.com/Nekland/Tools/blob/master/src/StringTools.php#L80 . And the dep is only for dev.

But I can remove it, simple strpos should be ok here.